lib/hammer_cli_foreman_docker/docker_manifest.rb

```diff
@@ -31,7 +31,7 @@ def extend_data(record)
       class InfoCommand < HammerCLIKatello::InfoCommand
         output do
           field :id, _("ID")
-          field :image_id, _("Image ID")
+          field :manifest_id, _("Manifest ID")
           field :size, _("Size")
```

Comment:
The information provided for a Docker manifest are different than what was previously provided for an image; therefore, we need to update the output.

Refer to : https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/docker_manifests/show.json.rabl for details on what is currently available for a manifest.